data/gui/themes/default/dialogs/wml_message.cfg

```diff
@@ -69,39 +61,32 @@ where
 ({___GUI_IMAGE_WIDTH})#enddef
 
 
-# This is the macro for the image width which is can be used as string
-# parameter.
+# This is the macro for the image width which can be used as string parameter.
 #define _GUI_IMAGE_WIDTH
 "{__GUI_IMAGE_WIDTH}"#enddef
```

Comment:
The "remove useless macro" commit ought to delete a few more lines here, and the `#undef` too.

Reply:
Right, thanks.